### PR TITLE
Rebuilded main README file

### PR DESCRIPTION
**Pull request by bogdanpolak**

### CODE REVIEW

**Comment by bogdanpolak** [comment]
Pls review - now everything is different - even MARS logo. Any kind of feedback is welcomed :-)

**Comment by bogdanpolak** [comment]
You can preview new README file in my repo

**Comment by andrea-magni** [comment]
Hi @bogdanpolak, I like the overall result. Just some notes:
1) I'd like to underline FireDAC integration is in the box but integrations against other DAC libraries is easy to implement (following the FireDAC example!)
2) The installation section of the readme file should be either more concise (just a link to the installation page) or more detailed (the whole content of the installation page)
3) I like the "Use the code Luke" section but somewhere there I would like to add a reference to the MARSCmd utility (that at the moment is the suggested way to bootstrap your first MARS project). Following your minimalistic approach I am also adding to the repo a new demo with a very minimal structure that can be used (through the MARSCmd again) as a template for your own projects.
4) The feature list should be much longer (I will try to fill the list with the most important features that actually are not listed anywhere at this very moment).
5) I like the idea of providing a "headline" version of the MARS logo... I will try to come up with a version from the original vectorized logo but with a "banner like" ratio.

Many thanks for the effort.
Let me know if you prefer me to accept this PR as is and then make some changes in my own repo or if you prefer to receive some of my consideration and then we can merge this PR.

Sincerely

**Comment by bogdanpolak** [comment]
Great! Then commit PR and fix some things in advance. You can also add some Issues if you need my support with that.

Ad.1) For sure (I just copy it form original repo)
Ad.2) Yeep! I can do more with that. Just add an issue. Or do it yourself if you like
Ad.3) Maybe, but at then moment MARScmd requires some improvements (my personal opinion) and it needs discussion. Then simple source code is better and easier to understand by the Delphi programmer (again my personal opinion)
Ad.4) For sure! I can copy some features form your slides and then you can add missing or fix what I will enter
Ad.5) No problem. I created this logo to show you the idea of size and the information, but be aware of size (now 21 KB). By the way, there are a lot of PNG files in the project that weigh a lot (it's important when you are cloning repo)

File: README.md [hunk 1]
```diff
@@ -1,44 +1,99 @@
-![MARS-curiosity logo](https://www.andreamagni.eu/images/MARS-Curiosity-d.png)
+![MARS-curiosity logo](media/logo-small-MARS.png)
 
-_\* The Delphi stylized helmet icon is trademark of Embarcadero Technologies._
```
Comment (baka0815) [inline README.md]: As you still have that image in the logo I think the comment should stay as to not infringe the copyright.

**Comment by bogdanpolak** [comment]
Now it's ready to commit

**Comment by bogdanpolak** [comment]
@andrea-magni any feedback on that ... if you like different solution this OK for me. Maybe you want me to change logo image to original one for some time. You can even close this PR if you prefer, but pls do not allow it to wait so long

**Pull request merged**